app/models/user.rb

```diff
@@ -7,10 +7,12 @@ class User < ApplicationRecord
   include DateHelper
 
   before_save :normalize_phone_number
+  before_create :set_token_expiration
   after_create :skip_email_confirmation_upon_creation
   after_create :create_preference_set
   before_update :record_previous_email
   has_secure_token :token, length: 36
+  has_secure_token :refresh_token, length: 36
```

Comment:
36 what? seconds, minutes, days, weeks?

Reply:
Vague syntax refers to token length, in this case. The token is 36 characters long.